cst.yml

```
@@ -63,7 +63,11 @@ fileExistenceTests:
     isExecutableBy: 'any'
 
 commandTests:
-  - name: "Check that `java` is present in the PATH and default to JDK17"
-    command: "java"
+  - name: "Check that `java` binary for agent processes is present and defaults to JDK17"
```

wording seems wrong, if it was the default you would be checking `java`
```suggestion
  - name: "Check that `java` 17 binary for agent processes is present"
```